swap_meet/electronics.py

```diff
@@ -1,2 +1,15 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+
+class Electronics(Item):
+    """Inherits from 'Item' class and creates an 'Electronics' object."""
+    
+    def __init__(self, category= "Electronics", **kwargs):
+    #https://stackoverflow.com/questions/66761435/child-class-override-parents-parameters
+        super().__init__(**kwargs)
+        self.category = category
```

Comment:
Rather than declaring `self.category` locally, I would recommend taking advantage of the inherited `__init__()` function and passing the `category` into that function like this:
```
super().__init__(category, condition, age)
```
Second note, is to be careful about having parameters for data that shouldn't change. It's great that you set `category` to have a default value of "Electronics", but that won't stop users for passing in a different value. To prevent users from changing the value of `category`, you can remove it from being a parameter and hardcode the value. It could look something like this:
```
def __init__(self, condition, age):
    super().__init__("Electronics", condition, age)
```
And now our users can't change the category. Yay! 😄